### ISSUE TITLE
Installation of kanboard.

### ISSUE THREAD

**Issue by hakancarlsson** [fguillot/kanboard#1620]
Aft installation of kanboard (http://kanboard.net/) I get this message:
Warning: require(/customers/1/e/0/cmproject.se/httpd.www/Kanban/app/common.php): failed to open stream: No such file or directory in /customers/1/e/0/cmproject.se/httpd.www/Kanban/index.php on line 4 Fatal error: require(): Failed opening required '/customers/1/e/0/cmproject.se/httpd.www/Kanban/app/common.php' (include_path='.:/usr/share/php') in /customers/1/e/0/cmproject.se/httpd.www/Kanban/index.php on line 4 .

Please rely at: hakan.carlsson@mail.com

**Comment by BlueTeck** [comment]
Which PHP version are you using? 
Did you install the required extensions? 
Which Kanboard Version are you installing? 
Zip from Kanboard.net or Github?

**Comment by hakancarlsson** [comment]
Hmm. I didnt see any extension.
I installed "kanboard-latest.zip".
I will try to install in local first and see if i can get it working.

Thanks for your reply.

**Comment by hakancarlsson** [comment]
Where CAN i find the required extension.?

**Comment by BlueTeck** [comment]
see documentation:

https://github.com/fguillot/kanboard/blob/master/doc/requirements.markdown

**Comment by hakancarlsson** [comment]
Thanks
I realised that I made an error when uploading.
I have deleted and uploaded by drag and drop at my webhosts server and now its correct.
When I press index.php I can login.

My last problem now is to link it to my homepage.
I have a subpages dedicated to kanbord www.http://cmproject.se/cm_intranet.html
But I'm not sure how the html-code should look like.
This is a page with login function but I think I now how to start a session.

Thanks to your help, I'm now close to getting a perfect kanboard-system on my website.
